crates/cli/cli/src/chainspec.rs

```diff
@@ -2,12 +2,40 @@ use std::sync::Arc;
 
 use clap::builder::TypedValueParser;
 
+#[derive(Debug, Clone)]
+struct Parser<C>(std::marker::PhantomData<C>);
+
+impl<C: ChainSpecParser> TypedValueParser for Parser<C> {
+    type Value = Arc<C::ChainSpec>;
```

Comment:
this should also override possible_values

Reply:
It causes duplicating data in help message (we also have values list in `chain_help()`) https://github.com/paradigmxyz/reth/actions/runs/10587216380/job/29337535172